src/js/extensions/paste.js

```diff
@@ -60,7 +60,10 @@
             [new RegExp(/<\/?o:[a-z]*>/gi), ''],
 
             // Microsoft Word adds some special elements around list items
-            [new RegExp(/<!\[if !supportLists\]>(((?!<!).)*)<!\[endif]\>/gi), '$1']
+            [new RegExp(/<!\[if !supportLists\]>(((?!<!).)*)<!\[endif]\>/gi), '$1'],
+
+            // remove font tag
+            [new RegExp(/<font[^><]*>|<.font[^><]*>/gi), '']
```

Comment:
Ok, my main ideia was to remove every formatted strings when paste with cleanPastedHTML.
What I'm trying to do is paste with no color, size or font type associated, just to match the existing style of the web app.
Maybe I'm not understanding what cleanPastedHTML means to you.